extra-network/netatalk/spec

```diff
@@ -0,0 +1,3 @@
+VER=3.1.12
+SRCTBL="https://versaweb.dl.sourceforge.net/project/netatalk/netatalk/3.1.12/netatalk-3.1.12.tar.gz"
```

Comment:
Please use the new ACBS spec file format.

Reply:
Now using Git repo